.github/workflows/build-test-javascript.jsonnet

```diff
@@ -98,7 +98,7 @@ local build_job =
 
 local test_job = {
   'runs-on': 'ubuntu-latest-16-core',
-  container: 'emscripten/emsdk:3.1.46',
+  container: 'emscripten/emsdk:3.1.51',
```

Comment:
maybe it was good adding a comment explaining 51 comes with this big llvm opti that makes a big
difference (and give the time saved).
Would be good also to factorize those flags in those 4 Makefiles.